client/components/jetpack/card/jetpack-product-card-alt-2/features-item.tsx

```diff
@@ -1,49 +1,115 @@
 /**
  * External dependencies
  */
+import classNames from 'classnames';
 import { isObject } from 'lodash';
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import { Button } from '@wordpress/components';
 import Gridicon from 'calypso/components/gridicon';
+import FeaturesProductSlideOut from './features-product-slide-out';
 import InfoPopover from 'calypso/components/info-popover';
 import { preventWidows } from 'calypso/lib/formatting';
+import { slugToSelectorProduct } from 'calypso/my-sites/plans-v2/utils';
+import {
+	FEATURE_TO_PRODUCT_ALT_V2,
+	FEATURE_TO_MONTHLY_PRODUCT_ALT_V2,
+} from 'calypso/my-sites/plans-v2/constants';
 
 /**
  * Type dependencies
  */
 import type { ProductCardFeaturesItem } from './types';
+import type { Duration, PurchaseCallback } from 'calypso/my-sites/plans-v2/types';
 
 export type Props = {
 	item: ProductCardFeaturesItem;
+	billingTerm: Duration;
+	onProductClick: PurchaseCallback;
 };
 
 type IconComponent = FunctionComponent< { icon: string; className?: string } >;
 
 const DEFAULT_ICON = 'checkmark';
 
-const JetpackProductCardFeaturesItem: FunctionComponent< Props > = ( { item } ) => {
-	const { icon, text, description, subitems } = item;
+const JetpackProductCardFeaturesItem: FunctionComponent< Props > = ( {
+	item,
+	billingTerm,
+	onProductClick,
+} ) => {
+	const { icon, text, description, isHighlighted, slug } = item;
 	const iconName = ( isObject( icon ) ? icon?.icon : icon ) || DEFAULT_ICON;
 	const Icon = ( ( isObject( icon ) && icon?.component ) || Gridicon ) as IconComponent;
+	const translate = useTranslate();
+
+	const [ slideOutExpanded, setSlideOutExpanded ] = useState( false );
+
+	const slideOutProductSlug =
+		billingTerm === 'TERM_ANNUALLY'
+			? FEATURE_TO_PRODUCT_ALT_V2[ slug ]
+			: FEATURE_TO_MONTHLY_PRODUCT_ALT_V2[ slug ];
+
+	const slideOutProduct = slugToSelectorProduct( slideOutProductSlug );
+
+	const toggleSlideOut = () => {
```

Comment:
Just a suggestion here: since `toggleSlideOut` is passed as a prop to `Button`, we could wrap it with `useCallback`, so that a new instance isn't created at each render. It wouldn't change anything here I believe, but I'd say it's good practice since it could potentially prevent re-renders (for memoized components).

Reply:
Yes, great suggestion, thank you 🙂